newsroom/web/default_settings.py

```diff
@@ -396,7 +396,7 @@
             "button_label": lazy_gettext("Completed"),
         },
     },
-    "agenda_top_story_sorting": False,
+    "agenda_top_story_sorting_only": True,
```

Comment:
would call it probably `agenda_sort_events_with_coverage_on_top` or something to be more descriptive